### DISCUSSION TITLE
.msixupload (or bundle .zip)

### DISCUSSION THREAD

**Discussion by myokeeh** [microsoft/msstore-cli#12] [Q&A]
How do I upload an .msixupload (or bundle .zip) with this?

**Comment by azchohfi** [comment]
Hello @myokeeh! Whay type of application are you trying to publish? UWP? WinAppSDK? Just trying to understand the scenario better.
Right now, we support only specific types of projects for uploading the bundle, but we are looking at specific scenarios to support loose msixupload files.

**Reply by myokeeh** [reply]
UWP, but WinAppSDK/WinUI3, too.

**Comment by zooba** [comment]
I have this same scenario. We currently produce `msixupload` files and manually upload them through Partner Center, but want to automate it. This seems like the closest possible tool for doing it, but as noted, there's currently no way to take an existing `msixupload` without also setting up some (unspecified?) "project configurator".

**Comment by brunvonlope** [comment]
In GIMP we managed to produce a .MSIXUPLOAD file (with .appxsym symbols included) fully automatic using only Windows SDK (makepri and makeappx). I was hoping to use this tool to not manually make submissions but I think that I misunderstood the scope of the tool. 

The msstore-cli tool can't really upload a MSIX (which should be the **the** Windows packaging format by excelence)? 😅
